### PR TITLE
Add ::spelling-error and ::grammar-error

### PR DESCRIPTION
**Pull request by foolip**

### CODE REVIEW

File: feature-group-definitions/spelling-grammar-error.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: "::spelling-error and ::grammar-error"
+description: The `::spelling-error` and `::grammar-error` CSS pseudo-elements match text that is highlighted as misspelled and grammatically incorrect, respectively.
```
Comment (ddbeck) [inline feature-group-definitions/spelling-grammar-error.yml:2]: "Highlighted" implies some visual styling to me. Maybe this?

```suggestion
description: The `::spelling-error` and `::grammar-error` CSS pseudo-elements match text that is identified as misspelled and grammatically incorrect, respectively.
```

**Review by ddbeck — APPROVED** [review]

File: feature-group-definitions/spelling-grammar-error.yml (hunk 1 above)
Comment (foolip) [reply feature-group-definitions/spelling-grammar-error.yml:2]: There is visual styling, and I think the main point of these pseudo-elements is to change or remove that styling.

Comment (foolip) [reply feature-group-definitions/spelling-grammar-error.yml:2]: (The browser isn't looking for these errors except when they will be highlighted, so this is really in form controls and other text input.)

Comment (ddbeck) [reply feature-group-definitions/spelling-grammar-error.yml:2]: OK, it's fine to leave this as is then. I'll merge this.

**Pull request merged**